lib/src/test/java/com/dynatrace/metric/util/MetricValuesTest.java

```diff
@@ -21,15 +21,23 @@
 class MetricValuesTest {
   @Test
   public void testFormatDouble() {
-    assertEquals("0", MetricValues.formatDouble(0));
-    assertEquals("0", MetricValues.formatDouble(-0));
-    assertEquals("0", MetricValues.formatDouble(.000000000000000));
-    assertEquals("1", MetricValues.formatDouble(1));
-    assertEquals("1", MetricValues.formatDouble(1.00000000000000));
-    assertEquals("1.234567", MetricValues.formatDouble(1.234567));
-    assertEquals("1.234568", MetricValues.formatDouble(1.234567890));
-    assertEquals("-1.234567", MetricValues.formatDouble(-1.234567));
-    assertEquals("-1.234568", MetricValues.formatDouble(-1.234567890));
+    assertEquals("0.0", MetricValues.formatDouble(0));
+    assertEquals("0.0", MetricValues.formatDouble(-0));
+    assertEquals("0.0", MetricValues.formatDouble(.000000000000000));
+    assertEquals("1.0", MetricValues.formatDouble(1));
+    assertEquals("1.0", MetricValues.formatDouble(1.00000000000000));
+    assertEquals("1.23456789", MetricValues.formatDouble(1.23456789000));
+    assertEquals("1.1234567890123457", MetricValues.formatDouble(1.1234567890123456789));
+    assertEquals("-1.23456789", MetricValues.formatDouble(-1.23456789000));
+    assertEquals("-1.1234567890123457", MetricValues.formatDouble(-1.1234567890123456789));
+    assertEquals("200.0", MetricValues.formatDouble(200));
+    assertEquals("200.0", MetricValues.formatDouble(200.00000000));
+    assertEquals("1.0E12", MetricValues.formatDouble(1_000_000_000_000d));
```

Comment:
```suggestion
    assertEquals("1.0E12", MetricValues.formatDouble(1_000_000_000_000d));
    assertEquals("1.0E10", MetricValues.formatDouble(1e10));
```
To make sure no trailing zeros are trimmed.